main.go

```diff
@@ -13,63 +13,35 @@ import (
 	"github.com/bitrise-io/go-steputils/cache"
 	"github.com/bitrise-io/go-steputils/command/gems"
 	"github.com/bitrise-io/go-steputils/command/rubycommand"
+	"github.com/bitrise-io/go-steputils/stepconf"
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-utils/pathutil"
-	"github.com/pkg/errors"
 )
 
 // ConfigsModel ...
 type ConfigsModel struct {
-	SourceRootPath  string
-	PodfilePath     string
-	Verbose         string
-	IsCacheDisabled string
+	Command         string `env:"command,opt[install,update]"`
+	SourceRootPath  string `env:"source_root_path,required"`
```

Comment:
It was checked to be a directory previously.
```suggestion
	SourceRootPath  string `env:"source_root_path,dir"`
```